pyramid_mailer/message.py

```diff
@@ -105,7 +101,7 @@ def to_mailbase(self, default_content_type=None):
 
         if not data:
             raise RuntimeError('No data provided to attachment')
-        
+
```

Comment:
My editor is stripping trailing white space... makes this a little messy

Reply:
That's fine.